migrations/tasks/rewards-flip-negative-costs.js

```diff
@@ -0,0 +1,57 @@
+// @migrationName = 'RewardsMigrationFlipNegativeCostsValues';
+// @authorName = 'hamboomger';
+// @authorUuid = '80b61b73-2278-4947-b713-a10112cfe7f5';
+
+/*
+ * For each reward with negative cost, make it positive
+ * by assigning it an absolute value of itself
+ */
+
+import { Task } from '../../website/server/models/task';
+
+async function flipNegativeCostsValues () {
+  const query = {
+    type: 'reward',
+    value: { $lt: 0 },
+  };
+
+  const totalCount = await _getTotalCount(query);
+  console.log(`Total documents to be changed: ${await Task.find().exec()}`);
```

Comment:
can you remove this line? Otherwise it'll try to load all tasks in the database :)

Reply:
Oopsie daisy, I couldn't run the migration tool properly so I tested how many documents there are at all (it showed 0), it seems I accidentally left it this way